Code/src/pipeline/__init__.py

```diff
@@ -599,33 +630,20 @@ def split_train_test_val(
             )
         self.split_type = split_type
 
-        # Allow `split_train_test_val` to default to using all columns
-        # for the set of `X_cols`
-        if X_cols is None:
-            X_cols = sorted(
-                list(
-                    set(self.df.columns) - set([
-                        # Remove identifying fields, as well as the
-                        # output `is_laundering`
-                        "from_account",
-                        "from_account_id",
-                        "from_account_idx",
-                        "from_bank",
-                        "is_laundering",
-                        "to_account",
-                        "to_account_id",
-                        "to_account_idx",
-                        "to_bank",
-                    ])
-                )
-            )
-        self.X_cols = X_cols
-        if not keep_account_idx:
-            self.X_cols = self.X_cols + ['from_account_idx','to_account_idx']
+        # Allow `split_train_test_val` to default to using all columns for the set of `X_cols`
+        cols = X_cols or self.df.columns
+         
+        # Remove identifying fields, as well as the output `is_laundering`
+        exclude_cols = {"from_bank", "to_bank", "from_account","to_account","from_account_idx", "to_account_idx","from_account_id","to_account_id","is_laundering"}
```

Comment:
```suggestion
        exclude_cols = {
            "from_bank",
            "to_bank",
            "from_account",
            "to_account",
            "from_account_idx",
            "to_account_idx",
            "from_account_id",
            "to_account_id",
            "is_laundering",
        }
```

PEP8 [asks](https://peps.python.org/pep-0008/#maximum-line-length) for <80 characters per line, nitpicking this one (but not every long line in our code base) because I find it easier to read the list of exclusions this way